challenges/objects-arrays.js

```diff
@@ -12,23 +12,52 @@
 
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
 
+const tRex = {
+  name: 'tyrannosaurus',
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous',
+
+  speak: function(){
+    console.log(`RAWERSRARARWERSARARARRRR!`);
```

Comment:
just something to keep in mind as you are building more projects. While we are using console.log's now to just show stuff is working you might want to start commenting that out and adding return statements to functions going forward